src/site/content/en/blog/performance-tool-score-diffs/index.md

```diff
@@ -0,0 +1,188 @@
+- 3X - First Contentful Paint (FCP)
+- 1X - First Meaningful Paint (FMP)
+- 2X - First CPU Idle (FCPUI)
+- 5X - Time to Interactive (TTI)
+- 4X - Speed index (SI)
+- 0X - Max Potential First Input Delay (MPFID)
+
+{% Aside %}
+To learn more see the _[Lighthouse variance 1-pager.](https://developers.google.com/web/fundamentals/performance/user-centric-performance-metrics)_
+{% endAside %}
+
+TMS uses only FCP (from both Lighthouse and from the CrUX) to provide both site and page speed scores.
+
+<div class="w-table-wrapper">
+  <table>
+    <thead>
+      <tr>
+        <th>Tool Name</th>
+        <th>Lab / Field Tool</th>
+        <th>Lab Metrics*</th>
+        <th>Field Metrics*</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><a href="https://www.thinkwithgoogle.com/intl/en-gb/feature/testmysite">Test My Site</a></td>
+        <td>Lab & Field</td>
+        <td>FCP**</td>
+        <td>FCP**</td>
+      </tr>
+      <tr>
+        <td><a href="https://developers.google.com/speed/pagespeed/insights/">PageSpeed Insights</a></td>
+        <td>Lab & Field</td>
+        <td>FCP, FMP, FCPUI, TTI, SI</td>
+        <td>FCP, FID</td>
+      </tr>
+      <tr>
+        <td><a href="https://developers.google.com/web/tools/lighthouse/">Lighthouse</a></td>
+        <td>Lab</td>
+        <td>FCP, FMP, FCPUI, TTI, SI</td>
+        <td>---</td>
+      </tr>
+    </tbody>
+    <caption>Performance metrics considered in score (by tool).</caption>
+  </table>
+
+  <small>* other metrics might be measured by the tools, but they aren’t included in performance/speed score calculations<br>** TMS uses FID as a tiebreaker in cases where FCP between compared sites is the same (but FCP is the lead metric)</small>
+</div>
+
+
+## Test My Site (TMS)
+TMS is a ‘Think with Google’ tool that allows you to compare your FCP to that of your competitors and estimate the monetary value of improving your site’s performance.
+
+### Mobile Site Speed: Field Data, FCP Only, Site Level
+The first part of the report, the mobile site speed seconds figure === First Contentful Paint (FCP) of the origin from CrUX.
+
+Site speed is the speed of **all the pages** that make up your site.
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="F1_tms_sitespeed.png" alt="Test my site mobile site speed score in seconds, with a 'Slow' rating and a 'No Change' monthly trend.">
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    Mobile Site Speed (Field Data, FCP Only, Site Level)
+  </figcaption>
+</figure>
+
+### Site Speed Tools: Field Data, FCP Only, Site Level
+In the second part of the report, the mobile site speed seconds figure === First Contentful Paint (FCP) of the origin from CrUX. 
+
```

Comment:
See general comment above-- I'm still lost here. Lots of abbreviations and assumptions around field and lab, etc.